eq-author-api/db/models/DynamoDB.js

```diff
@@ -33,58 +33,59 @@ const baseQuestionnaireSchema = {
   version: {
     type: Number,
   },
+  description: {
+    type: String,
+  },
+  legalBasis: {
+    type: String,
+  },
+  navigation: {
+    type: Boolean,
+  },
+  surveyId: {
+    type: String,
+  },
+  theme: {
+    type: String,
+  },
+  summary: {
+    type: Boolean,
+  },
+  sections: {
+    type: Array,
+    required: true,
+  },
+  metadata: {
+    type: Array,
+    required: true,
+  },
+  createdAt: {
+    type: Date,
+    required: true,
```

Comment:
We can probably go back to using Dynamoose `timestamps:true`

Reply:
I just tried and it seems to clash with our values causing it to merge. Maybe we can look at doing it later. Unless you think its worth digging into more?